src/services/ParticipantProjectSpeciesService.ts

```diff
@@ -62,9 +62,17 @@ const get = async (projectId: number, projectSpeciesId: number): Promise<Project
   });
 };
 
-const list = async (projectId: number): Promise<AllSpeciesResponse> => {
+const list = async (projectId?: number): Promise<AllSpeciesResponse> => {
   return new Promise((resolve) => {
-    resolve({ data: { projectSpecies: mockProjectSpecies?.[projectId] || [] }, requestSucceeded: true });
+    if (projectId === undefined) {
+      const allProjectsSpecies: ProjectSpecies[] = [];
+      Object.values(mockProjectSpecies).forEach((speciesList) => {
+        allProjectsSpecies.push(...speciesList);
+      });
+      resolve({ data: { projectSpecies: allProjectsSpecies }, requestSucceeded: true });
```

Comment:
Thanks for doing this! Could we also make the projectId optional in the thunk?

Reply:
Oh yeah oops! Sorry will push the change up ASAP.

Reply:
Project ID needs to be required. Participant Project Species exist in addition to Organization Species and are accessed through the project ID